napari/_vispy/vispy_base_layer.py

```diff
@@ -176,6 +176,8 @@ def _transform_position(self, position):
             position = transform.map(list(position))[
                 : len(self.layer.dims.displayed)
             ]
+            # Offset position so that pixel center is at 0
+            position = position - 0.5
```

Comment:
Maybe just add `- 0.5` to the line above? I generally dislike overwriting existing names.

Reply:
done